docs/source/frameworks/fastai.rst

```diff
@@ -0,0 +1,246 @@
+=======
+fast.ai
+=======
+
+.. note::
+
+   BentoML requires fastai version 2 or higher to be installed. We do not currently support version 1.
+
+fastai is a deep learning library which provides both high-level components for practioners to get SOTA results in standard deep learning domains, and low-level components
+for researchers to build new approaches. The abstractions can be achieved by leveraging the flexibility of the PyTorch library and its ecosystem (PyTorch Lightning, Catalyst, etc.)
+
+fastai also provides a great sets of `documentation <docs.fast.ai>`_ and
+`migration guides <https://docs.fast.ai/#Migrating-from-other-libraries>`_ from
+PyTorch-related library.
+
+
+Fine tuning the language model
+------------------------------
+
+.. seealso::
+
+   This section is based heavily on `Transfer Learning with text <https://docs.fast.ai/tutorial.text.html#The-ULMFiT-approach>`_ from fastai.
```

Comment:
Let's link to the general model training documentation of fastai and ask the readers to refer to it.

Reply:
I thinking linking to one big blocks provides a quick value for users to see how the models can be created. Obviously we will still keep the link as the main reference.